spec/controllers/spotlight/catalog_controller_spec.rb

```diff
@@ -111,6 +111,29 @@
         expect(json['docs'].first['title']).to eq 'KAART der REYZE van drie Schepen naar het ZUYDLAND in de Jaaren 1721 en 1722'
       end
     end
+
+    describe 'GET manifest' do
+      context 'document is an uploaded resource' do
+        it 'returns the json manifest produced by Spotlight::IiifManifestPresenter, based on the retrieved document and the controller' do
+          allow_any_instance_of(::SolrDocument).to receive(:uploaded_resource?).and_return(true)
+          response_hash = { field: 'value' }
+          iiif_manifest_presenter = double(Spotlight::IiifManifestPresenter, iiif_manifest_json: response_hash.to_json)
+          expect(Spotlight::IiifManifestPresenter).to receive(:new).with(an_instance_of(::SolrDocument), controller).and_return(iiif_manifest_presenter)
+
+          get :manifest, params: { exhibit_id: exhibit, id: '1-1' }
+          expect(response).to be_successful
+          expect(response.body).to eq(response_hash.to_json)
+        end
+      end
+      context 'document is not an uploaded resource' do
+        it 'returns a 404 when called on something other than an uploaded resource' do
+          allow_any_instance_of(::SolrDocument).to receive(:uploaded_resource?).and_return(false)
```

Comment:
Did you find that this was necessary?  I wonder if just passing an id to a solr document fixture that is not an uploaded resource should just do the right thing.